app/src/main/kotlin/org/fossify/contacts/activities/ViewContactActivity.kt

```diff
@@ -46,6 +48,19 @@ class ViewContactActivity : ContactActivity() {
     private val binding by viewBinding(ActivityViewContactBinding::inflate)
 
     companion object {
+
+        fun getCountryIso(): String {
+            val locale = Locale.getDefault()
+            return locale.country
+        }
+        fun formatPhoneNumber(phoneNumber: String): String {
+            if (phoneNumber.length > 4){
+                return PhoneNumberUtils.formatNumber(phoneNumber, getCountryIso()).toString()
+            } else {
+                return phoneNumber
+            }
+        }
+
```

Comment:
This function shouldn't be here because it's also used in ContactsAdapter. Do not unnecessarily tie different things together i.e. do not do `ViewContactActivity.formatPhoneNumber()` in `ContactsAdapter`. When you have a reusable function, place it in some place common like a utility class or an extension function. 

Please convert it to a String extension function in the `extensions` package. Example:

```kotlin
fun String.formatPhoneNumber(minimumLength : Int = 4) {
     val country = Locale.getDefault().country
     // format phone number...
     return formattedPhoneNumber
}
```

Ideally, this extension function would go into the commons library but I guess I'll do that later.